.travis.yml

```diff
@@ -14,9 +14,14 @@ matrix:
     - os: linux
       env: CUDA=8.0.61-1 CMAKE_OPTIONS="-DPENGUINV_BUILD_TEST=ON"
       before_install: source script/travis/install_cuda.sh
-    - os: osx
+    - os: linux
+      env: QT_BASE=59 CMAKE_OPTIONS="-DPENGUINV_BUILD_QT_EXAMPLE=ON"
+      before_install: source script/travis/install_qt5.sh
```

Comment:
Did you find out what's the difference between `install` and `before_install`? :)

Reply:
I found a very good explanation [here](https://github.com/travis-ci/travis-ci.github.com/pull/333).
TL;DR
- before_install: installs system dependencies before even changing into repo dir (e.g. apt-get update etc...)
- install: installs prerequisites or dependencies necessary to run the build

Reply:
Awesome.